src/handlers/walletReadyEvents.js

```diff
@@ -65,6 +65,7 @@ export const runWalletBackupStatusChecks = () => {
     setTimeout(() => {
       logger.log('showing BackupSheet with existing_user step');
       Navigation.handleAction(Routes.BACKUP_SHEET, {
+        single: true,
```

Comment:
There are other occurrences where we navigate to `BACKUP_SHEET`.  Do you need to add the `single` param there too?

- `AlreadyBackupView.js`
- `NeedsBackupView.js`
- `redux/data.js`
- `ImportSeedPhraseSheet.js`

Reply:
fixed

Reply:
It's not expected in AlreadyBackupView and ImportSeedPhraseSheet